### PR TITLE
Add return user FTUE for the library menu.

### PR DESCRIPTION
**Pull request by yipingliao**
@2is10 Not activated yet; needs backend support.

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/app.js [hunk 1]
```diff
@@ -122,6 +122,15 @@ angular.module('kifi', [
       $scope.libraryMenu = {
         visible: false
       };
+
+      if (false) {  // Needs condition from backend.
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/app.js:130]: Set this to true for testing. What it should be is something like `if (profileService.me.showReturnLibMenuFTUE)`'.

File: kifi-backend/modules/shoebox/angular/src/layout/libraryMenu/libraryMenu.js [hunk 2]
```diff
@@ -270,6 +270,7 @@ angular.module('kifi')
         scope.$watch('libraryMenu.visible', function (visible) {
           if (visible) {
             openMenu();
+            scope.libraryMenuHelp.visible = false;
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/layout/libraryMenu/libraryMenu.js:273]: Close FTUE on library menu open.

File: kifi-backend/modules/shoebox/angular/src/layout/libraryMenu/libraryMenuHelp.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      link: function (scope , element /*, attrs*/) {
+        function setLeftPosition() {
+          element.css({'left': angular.element('.kf-lih-toggle-menu').offset().left - 13 + 'px'});
+        }
+
+        scope.$watch('libraryMenuHelp.visible', function (visible) {
+          if (visible) {
+            setLeftPosition();
+          } else {
+            // Let the server know that the user has seen this menu.
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/layout/libraryMenu/libraryMenuHelp.js:20]: Needs backend endpoint here.

File: kifi-backend/modules/shoebox/angular/src/app.js [hunk 4]
```diff
@@ -122,6 +122,15 @@ angular.module('kifi', [
...
+        $timeout(function () {
+          $scope.libraryMenuHelp = {
+            visible: true
+          };
+        }, 2000);
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/app.js:133]: Wait for 2 seconds for the user to realize that something is different before showing the pop-up.

**Comment by 2is10** [comment]
Great, thanks.

**Pull request merged**